connectivity/drivers/ble/FEATURE_BLE/COMPONENT_BlueNRG_2/README.md

```diff
@@ -0,0 +1,119 @@
+# BlueNRG_2
+
+HCI driver for BlueNRG_2 BLE component
+
+A simple table like this could help:
+
+|Module Name|Processor Name|Bluetooth compliance|Status|Used in shields & boards|Link|
+|-------------|-----------|-----|-|-|-|
+|SPBTLE-RF    |BlueNRG-MS (network processor) |v4.1 |Not recommended for new designs             |X-NUCLEO-IDB05A1, DISCO-L475VG-IOT01A, DISCO-L562QE | https://www.st.com/en/wireless-transceivers-mcus-and-modules/spbtle-rf.html |
+|BlueNRG-M0   |BlueNRG-MS (network processor) |v4.2 |Active (included in ST's Longevity Program) |X-NUCLEO-IDB05A2 | https://www.st.com/en/wireless-transceivers-mcus-and-modules/bluenrg-m0.html |
+|BlueNRG-M2SP |BlueNRG-2 (application processor) |v5.0 |Active (included in ST's Longevity Program) |X-NUCLEO-BNRG2A1 | https://www.st.com/en/wireless-transceivers-mcus-and-modules/bluenrg-m2.html |
+
+
+The library uses ARM Cordio stack. It does not work with the stock firmware that is loaded in the BLE module of the X-NUCLEO-BNRG2A1 expansion board.
+
+In order to use this library with X-NUCLEO-BNRG2A1, you need to replace the firmware of the BLUENRG-M2SP BLE module mounted on that expansion by using one of these binaries (provided for both STM32 NUCLEO-L476RG and STM32 NUCLEO-F401RE):
+
+[BlueNrg2FwUpdater_L4.bin](fwupdater/BlueNrg2FwUpdater_L4.bin)
+
+[BlueNrg2FwUpdater_F4.bin](fwupdater/BlueNrg2FwUpdater_F4.bin)
+
+Before flashing your target board with the respective binary, be careful to properly set the SPI reset on pin **D7** as reported in the picture below:
+
+![D7_SPI_rst](img/D7_SPI_rst.jpg)
+
+At the end of the updating procedure, the LED labeled as **LD2** on the STM32 NUCLEO board will be blinking.
+
+If, for some reason, the updating procedure does not succeed you can use the PC utility included in the [STSW-BNRGFLASHER](https://www.st.com/content/st_com/en/products/embedded-software/wireless-connectivity-software/stsw-bnrgflasher.html) package along with the [ST-LINK/V2](https://www.st.com/content/st_com/en/products/development-tools/hardware-development-tools/hardware-development-tools-for-stm32/st-link-v2.html) programmer.
```

Comment:
I'm trying to follow that. I can connect the board with an st link and read the memory and mass erase but flashing fails:
```
******FLASH OPERATION******
Connected N.1 device by ST Link
14:52:35.506: Waiting device to flash -> Device connected
14:52:35.641: Device Identification -> FAILED
14:52:35.779: Device Flash Programming -> FAILED
14:52:35.845: Device Disconnection ->FAILED
14:52:35.930: Flash operation finished!
******END FLASH OPERATION******
```
I'm using BLUENRG-X flasher utility v 4. Do I have to use some specific version for it to work?